packages/datadog-plugin-pino/test/index.spec.js

```diff
@@ -53,6 +54,25 @@ describe('Plugin', () => {
             expect(record).to.have.deep.property('msg', 'message')
           })
         })
+
+        if (!semver.intersects(version, '>=4.10.0 <=5.0.0-rc.1')) {
```

Comment:
Why these versions specifically? What is different in that range?

Reply:
In this small range (4.10.0 - 4.19 + 5.0.0-rc.1), there is a check in pino itself that goes `if (prettyPrint && outputStream !== process.stdout) throw`. So it makes it tedious to test anything as I'd have to mess with the global stdout. Do you think it's worth instrumenting the global `process.stdout` in the unit tests just for that ?

Reply:
Yeah, it's easy enough to stub/unstub it with Sinon that I think it's worth making sure that the test versions match the support range.